doc/cephadm/mon.rst

```diff
@@ -90,6 +99,9 @@ hosts to the cluster. No further steps are necessary.
 
     ceph orch apply mon label:mon
 
+Deploying Monitors Manually 
```

Comment:
I probably wouldn't call this "manual". See https://docs.ceph.com/en/latest/cephadm/troubleshooting/#manually-deploying-a-mgr-daemon for manually deploying a MGR daemon. 

I think the main difference here is: We're explicitly specifying the MON network here in this section

Reply:
Roger that. I've changed the title of this section to "Deploying Monitors on a Particular Network".